src/actions/verify.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  APIInteractionResponse,
+  ButtonStyle,
+  ComponentType,
+  InteractionResponseType,
+  MessageFlags,
+} from '@collabland/discord'
+import { DiscordActionRequest } from '@collabland/discord'
+
+import { config } from '@/config'
+
+export const handleVerifyAction = async (interaction: DiscordActionRequest) => {
+  const response: APIInteractionResponse = {
+    type: InteractionResponseType.ChannelMessageWithSource,
+    data: {
+      // https://discord.com/developers/docs/resources/channel#message-object-message-flags
+      flags: MessageFlags.Ephemeral,
+      embeds: [
+        {
+          title: `Welcome to ${interaction?.actionContext?.guildName}!`,
+          description: `To authenticate your identity through Rarimo's Proof of Humanity and Collab.Land Bot, please follow these steps:`,
+          fields: [
+            {
+              name: '1. Acquiring Rarimo Proof of Humanity',
+              value: `Click on the "Verify Your Humanity" button located directly beneath this message. This will redirect you to the Rarimo Proof of Humanity dApp, where you can obtain your proof of humanity. `,
```

Comment:
```suggestion
              value: `Click on the "Verify Your Humanity" button directly beneath this message. This will redirect you to the Rarimo Proof of Humanity DApp, where you can get verified. `,
```